cloud.ks

```diff
@@ -33,8 +35,7 @@ logvol / --size=3000 --fstype="xfs" --name=root --vgname=coreos
 
 ostreesetup --nogpg --osname=rhcos --remote=rhcos --url=@@OSTREE_INSTALL_URL@@ --ref=@@OSTREE_INSTALL_REF@@
 
-
-reboot
+poweroff
```

Comment:
Minor: this is fine, though we shouldn't actually need this if we have `--noreboot`, right?

Reply:
I added this while debugging `virt-install`.  It polls the domain being active, which I thought was being racy, but the way it actually works is that it tells libvirt to shutdown the domain on reboot, so it doesn't matter what we do here.